pokemongo_bot/cell_workers/sniper.py

```diff
@@ -0,0 +1,370 @@
+from __future__ import unicode_literals
+
+import os
+import time
+import json
+import requests
+
+from random import uniform
+from pokemongo_bot import inventory
+from pokemongo_bot.inventory import Pokemon, Pokemons
+from pokemongo_bot.base_task import BaseTask
+from pokemongo_bot.base_dir import _base_dir
+from pokemongo_bot.constants import Constants
```

Comment:
Not sure. Havent changed this. Just kept the old imports. I might take it off, if its not really used, though.